examples/federated-learning/tff/nvflare/docker-image/Dockerfile

```diff
@@ -0,0 +1,11 @@
+#checkov:skip=CKV_DOCKER_2: No need for HEALTHCHECK
+FROM nvflare/nvflare:2.4.1
+
+RUN useradd -ms /bin/bash nvflare && chown -R nvflare /opt/NVFlare
```

Comment:
Add two `ARG` to set the user id (uid) and the group id (gid), and set their defaults to 1000, as you seem to require in the deployment, because the uid and gid you get when creating a new user depend on the environment you create them in.

Then, use these uid and gid explicitly when creating the nvflare user.

This way, you give users the chance to customize them if they need to map this user to a specific one on the host.

Reply:
Done

Reply:
where? :)